src/Tests/EmsApi.Tests.csproj

```diff
@@ -1,19 +1,19 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>net5.0</TargetFramework>
+    <TargetFramework>net6.0</TargetFramework>
```

Comment:
Why not go to 7?

Reply:
I like to keep this the same version as what we use for NSwag so people don't need multiple sdks installed, and working through getting NSwag updated to 7 is going to be a heavier lift